src/main/java/org/xstefank/check/TemplateChecker.java

```diff
@@ -91,4 +93,9 @@ private static FormatConfig readConfig() {
         }
     }
 
+    private boolean shouldSkip(JsonNode payload) {
+        SkipChecker skipChecker = new SkipChecker(config.getFormat().getSkipPatterns(),payload);
+        return skipChecker.shouldSkip();
```

Comment:
I'll leave this to your judgement, do you think that the instance creation of the `SkipChecker` is necessary here? It will be created only for this one call and then eligible for garbage collection. Can you find some other way?